### ISSUE TITLE
The Group Media Gallery is not visible

### ISSUE THREAD

**Issue by abertranb** [buddydev/mediapress#19]
Hello there is a typo mistake in groups (in mediapress/modules/groups/loader.php line 36)

mpp_check_group_access
and should be
mpp_check_groups_access

Because in functions.php the function name is mpp_check_groups_access

```
function mpp_group_init() {

    mpp_register_status( array(
        'key'           => 'groupsonly',
        'label'         => __( 'Group Only', 'mediapress' ),
        'labels'        => array( 
                                'singular_name' => __( 'Group Only', 'mediapress' ),
                                'plural_name'   => __( 'Group Only', 'mediapress' )
        ),
        'description'   => __( 'Group Only Privacy Type', 'mediapress' ),
        'callback'      => 'mpp_check_group_access'
));
}
```

Best regards

**Comment by sbrajesh** [comment]
@abertranb 
Thank you for pointing. It was a mistake. Have updated on my local dev. Will mark this ticket closed when I update it here.

**Comment by sbrajesh** [comment]
Fixed in 162cc71abd60
